ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -528,6 +620,10 @@ public override void Dispatch(RCTEventEmitter eventEmitter)
         class ScrollViewerData
         {
             public ScrollMode HorizontalScrollMode = ScrollMode.Disabled;
+
+            public bool IsControlLoaded = false;
```

Comment:
>false [](start = 42, length = 5)

I doubt C# is in a hurry to change default(bool) to true - I like to make people think about default behaviors when reading code ;-) - you can either leave as is or remove, I don't care.

Reply:
fixed.

---
In reply to: [155706327](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r155706327) [](ancestors = 155706327)